src/lib/mina_base/zkapp_command.ml

```diff
@@ -365,47 +393,59 @@ module Call_forest = struct
         ; stack_hash = ()
         } )
 
-  let rec of_parties_list_map ~(f : 'p1 -> 'p2) ~(party_depth : 'p1 -> int)
-      (parties : 'p1 list) : ('p2, unit, unit) t =
-    match parties with
+  let rec of_zkapp_command_list_map ~(f : 'p1 -> 'p2)
```

Comment:
nit: maybe this would be better named `of_account_updates_list_map` (and similar `of_account_updates_list`, `to_account_updates_list`, etc)?

Reply:
`account_updates` already suggests it's a list, so I'm going to use `account_updates_map`, and so on.